UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
+        BrowserUtils.resetToAboutHome(tester())
+    }
+
+
+    override func beforeAll() {
```

Comment:
Hm, why do we need to do this? Every test should restore itself to the startup state once it finishes, so this test should already be in the default state at the beginning. If that's not the case, we should fix whatever test is running before this one.